cs1_lab02.Rmd

```diff
@@ -0,0 +1,91 @@
+---
+number: 2
+course: Computer Science 1
+material: Instruction 2
+---
+
+## 1.
+Write a program using a function which prints:
+```c++
+#include <stdio.h>
+
+void FunView();          // declaration of a function: line ended with semicolon!
+
+int main()               // main function
+{
+    FunView();           // call the function FunView
+    
+    return 0;
+}
+                         // definition of the function FunView
+void FunView()           // function header: contains return type and name
+{
+    printf("Hello!");    // function starting bracket
+}                        // function ending bracket
+```
+
+## 2.
+Modify the program by adding three new functions.
+First function should increment the global variable `Liczba` by $5$, e.g.:
```

Comment:
I feel like using an English variable name would be more friendly to non-Polish native speakers.

Reply:
Corrected.